operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -137,6 +137,19 @@
       + "the directory configured in the WebLogic domain configuration. Defaults to true.")
   private Boolean httpAccessLogInLogHome;
 
+  /**
+   * Full path of an optional liveness probe custom script for WebLogic server pods. This optional field
+   * is for advanced usage only and its value is not set by default. If the script specified by the value
+   * of this field is not found, then it is ignored and existing liveness probe script will perform
+   * its normal checks.
```

Comment:
Can this text be changed to match the attribute description in domain-resource.md?  We want a 'kubectl explain' to be fully helpful...